### PR TITLE
[wip] Potential changes to push upstream

### PR DESCRIPTION
**Pull request by SimenB**
Don't merge, we'll propose these changes upstream.

I already have an open PR upstream to allow us to use `error`, `warn` and `off`, instead of `2`, `1` and `0`

Btw I don't have write access to this repo, so had to fork it.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,7 +8,9 @@ module.exports = {
     ],
 
     rules: {
-        'eqeqeq': [ERR, 'smart']
+        'eqeqeq': [ERR, 'smart'],
+        'no-eq-null': OFF,
```
Comment (SimenB) [inline index.js]: direct competition with the above one

**Comment by SimenB** [comment]
PR opened: schibsted/eslint-config-spt#8

**Comment by sveisvei** [comment]
If we dont get a reponse soon, i vote for merging and backtracking upstream changes later.

File: index.js [hunk 2]
```diff
@@ -7,6 +8,8 @@ module.exports = {
...
+        'max-nested-callbacks': OFF
```
Comment (gregersrygg) [inline index.js]: Move to a test config?

Comment (SimenB) [inline index.js]: so it becomes `extends: 'finn/test'` for the consumer?

**Comment by SimenB** [comment]
Updated to remove max-callbacks.

We wanna merge this now, and rather remove some rules again if they're accepted upstream

**Pull request closed without merging**